.github/workflows/lint-pr.yml

```diff
@@ -4,20 +4,26 @@ on:
   pull_request:
     branches: [main]
 
+
+env:
+  UV_SYSTEM_PYTHON: 1
+
```

Comment:
same q

Reply:
ref. https://docs.astral.sh/uv/guides/integration/github/#using-uv-pip

```
If using the uv pip interface instead of the uv project interface, uv requires a virtual environment by default. To allow installing packages into the system environment, use the --system flag on all uv invocations or set the UV_SYSTEM_PYTHON variable.

The UV_SYSTEM_PYTHON variable can be defined in at different scopes.

Opt-in for the entire workflow by defining it at the top level:
```